consensus/e2e_tests/pacemaker_test.go

```diff
@@ -43,89 +43,24 @@ func TestPacemakerTimeoutIncreasesRound(t *testing.T) {
 
 	// Advance time by an amount shorter than the pacemaker timeout
 	advanceTime(t, clockMock, 10*time.Millisecond)
-
-	// Verify consensus started - NewRound messages have an N^2 complexity.
-	numExpectedMsgs := numValidators * numValidators
-	_, err := WaitForNetworkConsensusEvents(t, clockMock, eventsChannel, consensus.NewRound, consensus.Propose, numExpectedMsgs, consensusMessageTimeoutMsec, true)
-	require.NoError(t, err)
-
-	for pocketId, pocketNode := range pocketNodes {
-		assertNodeConsensusView(t, pocketId,
-			typesCons.ConsensusNodeState{
-				Height: 1,
-				Step:   uint8(consensus.NewRound),
-				Round:  0,
-			},
-			GetConsensusNodeState(pocketNode))
-	}
+	height := uint64(1)
+	step := uint8(consensus.NewRound)
+	round := uint8(0)
```

Comment:
Remove these 3 variables and make it explicit in the code.

If I look at

```
_ = waitForNewRound(t, clockMock, eventsChannel, pocketNodes, height, step, round+1, numValidators, consensusMessageTimeout, true)
```

I don't know what `height`, `step`, `round` are.

If I look at

```
_ = waitForNewRound(t, clockMock, eventsChannel, pocketNodes, 1, NewRound,1, numValidators, consensusMessageTimeout, true)
```

It becomes really obvious.

There's no "rule" for this. Just pretend you don't know anything and try to figure out what makes it easier to understad.